src/stylesheets/modules/_SiteNavigation.scss

```diff
@@ -5,19 +5,11 @@
   justify-content: space-between;
 
   &-item {
-    padding: 10px 10px;
+      font-weight: bold;
```

Comment:
SiteNavigation-item is a wrapper for link.
And you want to get bold text.
Text also is in the link.
So replace styles to correct class please